scripts/install-multi-user.sh

```diff
@@ -387,19 +387,25 @@ EOF
     fi
 
     for profile_target in "${PROFILE_TARGETS[@]}"; do
+        # TODO: I think it would be good to accumulate a list of all
+        #       of the copies so that people don't hit this 2 or 3x in
+        #       a row for different files.
```

Comment:
I would, to be frank, like it to work as commented, and not as coded. 

I didn't want to let that pass completely undocumented--but I'm chasing other white whales at the moment and do not have the emotional/intellectual energy to make this happen without an installer test suite that ensures the important bits stay nailed down.